frontend/src/js/controllers/authCtrl.js

```diff
@@ -55,6 +55,22 @@
             vm.wrnMsg = {};
         }
 
+        vm.passwordCheck = function() {
+            var parameters = {};
```

Comment:
no need to define this variable as you will pass passwords as function's argument